pkg/operator/set_operator_split.go

```diff
@@ -0,0 +1,197 @@
+package operator
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/internal/common"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/internal/common/flags"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/operator/split"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/rewards"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/telemetry"
+	"github.com/Layr-Labs/eigenlayer-cli/pkg/utils"
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients/elcontracts"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	eigenSdkUtils "github.com/Layr-Labs/eigensdk-go/utils"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/urfave/cli/v2"
+)
+
+func SetOperatorSplitCmd(p utils.Prompter) *cli.Command {
+	var operatorSplitCmd = &cli.Command{
+		Name:  "set-rewards-split",
+		Usage: "Set operator rewards split",
+		Action: func(cCtx *cli.Context) error {
+			return SetOperatorSplit(cCtx, p)
+		},
+		After: telemetry.AfterRunAction(),
+		Flags: getSetOperatorSplitFlags(),
+	}
+
+	return operatorSplitCmd
+}
+
+func SetOperatorSplit(cCtx *cli.Context, p utils.Prompter) error {
+	ctx := cCtx.Context
+	logger := common.GetLogger(cCtx)
+
+	config, err := readAndValidateSetOperatorSplitConfig(cCtx, logger)
+	if err != nil {
+		return eigenSdkUtils.WrapError("failed to read and validate operator split config", err)
+	}
+
+	cCtx.App.Metadata["network"] = config.ChainID.String()
+
+	ethClient, err := ethclient.Dial(config.RPCUrl)
+	if err != nil {
+		return eigenSdkUtils.WrapError("failed to create new eth client", err)
+	}
+
+	if config.Broadcast {
+
+		eLWriter, err := common.GetELWriter(
+			config.OperatorAddress,
+			config.SignerConfig,
+			ethClient,
+			elcontracts.Config{
+				RewardsCoordinatorAddress: config.RewardsCoordinatorAddress,
+			},
+			p,
+			config.ChainID,
+			logger,
+		)
+
+		if err != nil {
+			return eigenSdkUtils.WrapError("failed to get EL writer", err)
+		}
+
+		logger.Infof("Broadcasting set operator transaction...")
+
+		var receipt *types.Receipt
```

Comment:
You don't need to define this. In the next line you can just do ":="

Reply:
thanks, these were from the copied over code where we had to iterate over multiple claims, missed updating these